Makefile

```diff
@@ -43,6 +46,7 @@ lint: clean
 
 # used in development
 dev-setup:
+	$(VENV)
```

Comment:
This 👇 is the correct form of "dependent"

```suggestion
dev-setup: $(VENV)
```

Reply:
Oh, I see. Corrected.